src/main/java/response/HttpResponseWriter.java

```diff
@@ -0,0 +1,20 @@
+package com.td.HttpServer;
+import java.io.IOException;
+
+public class HttpResponseWriter {
```

Comment:
Same thing here. This class seems like a very simple delegator. Can you justify it's existence?